src/DIRAC/Resources/Storage/StorageElement.py

```diff
@@ -1405,5 +1404,93 @@ def addAccountingOperation(self, lfns, startDate, elapsedTime, storageParameters
         if not accRes["OK"]:
             self.log.error("Could not send accounting report", accRes["Message"])
 
+    def addMonitoringOperation(self, lfns, elapsedTime, storageParameters, callRes):
```

Comment:
There is quite some overlapping between this method and `def addAccountingOperation` method in this module. Can the two be merged?